### PR TITLE
README: Add a better description and new usages

### PR DESCRIPTION
**Pull request by Hywan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +29,138 @@ install](http://hoa-project.net/Source.html).
...
-…
+We propose a quick overview of how to use generic exceptions, how to listen all
+thrown exceptions through events and how to use group of exceptions.
+
+### Generic exceptions
+
+An exception is constitued of:
+  * A message,
+  * A code (optional),
+  * A list of arguments for the message (à la `printf`, optional),
```
Comment (Metalaka) [inline README.md:40]: à la française ? ;)

Comment (Hywan) [inline README.md:40]: « à la » is very classy :-p.

Comment (Metalaka) [inline README.md:40]: Ok, I didn't know that English people say it.

Comment (Hywan) [inline README.md:40]: @Metalaka https://en.wiktionary.org/wiki/%C3%A0_la

**Pull request merged**